java/ext/javadoctest.py

```diff
@@ -105,6 +105,7 @@ def setup(app) -> Dict[str, Any]:
     app.add_directive("testoutput", TestoutputDirective)
     app.add_builder(JavaDocTestBuilder)
     # this config value adds to sys.path
```

Comment:
The comment now is with the wrong line, can you move it?

Reply:
> The comment now is with the wrong line, can you move it?

Please note that all configurations below are subject to this comment.

Reply:
No: the comment is talking about `doctest_path` specifically.

Reply:
I will update them now